x-pack/solutions/security/packages/kbn-scout-security/src/playwright/fixtures/types.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  ScoutParallelTestFixtures,
+  ScoutParallelWorkerFixtures,
+  ScoutTestFixtures,
+  ScoutWorkerFixtures,
+} from '@kbn/scout';
+import { BrowserAuthFixture } from '@kbn/scout/src/playwright/fixtures/test/browser_auth';
+import { SecurityPageObjects } from './test/page_objects';
+import { DetectionRuleFixture } from './worker/apis/detection_rule';
+
+export interface SecurityBrowserAuthFixture extends BrowserAuthFixture {
+  loginAsPlatformEngineer: () => Promise<void>;
+}
+export interface SecurityTestFixtures extends ScoutTestFixtures {
+  browserAuth: SecurityBrowserAuthFixture;
+  pageObjects: SecurityPageObjects;
+}
+
+export interface SecurityWorkerFixtures extends ScoutWorkerFixtures {
+  detectionRuleApi: DetectionRuleFixture;
+}
```

Comment:
```
export interface SecurityApiServicesFixture extends ApiServicesFixture {
  detectionRuleApi: DetectionRuleApiService;
}

export interface SecurityWorkerFixtures extends ScoutWorkerFixtures {
  apiServices: SecurityApiServicesFixture;
}
```

Reply:
my bad, small correction on the property name: `detectionRuleApi` => `detectionRule`

Call in test spec:
```
apiServices.detectionRule.doSmth();
```